lib/spack/spack/cmd/ci.py

```diff
@@ -93,6 +93,13 @@ def setup_parser(subparser):
         help="process up-to-date specs\n\n"
         "generate jobs for specs even when they are up-to-date on the mirror",
     )
+    generate.add_argument(
+        "--prune-externals",
```

Comment:
Should there be a reciprocal of this, like `--no-prune-externals`

Reply:
Sure, that seems reasonable.